perf-tests/clusterloader2/pkg/imagepreload/imagepreload.go

```diff
@@ -36,7 +36,7 @@ import (
 
 const (
 	informerTimeout = time.Minute
-	manifest        = "$GOPATH/src/k8s.io/perf-tests/clusterloader2/pkg/imagepreload/manifests/daemonset.yaml"
+	manifest        = "$GOPATH/src/k8s.io/arktos/perf-tests/clusterloader2/pkg/imagepreload/manifests/daemonset.yaml"
```

Comment:
Can this be changed to "$GOPATH/src/arktos/perf-tests/... ?

Reply:
This change needs to work with PR 583 to get Promethus data, correct? Can they be merged?

Reply:
For the first comment, "$GOPATH/src/k8s.io/arktos/" is the correct path to the arktos repo in perf-test env. You should run into some problems if you put it as $GOPATH/src/arktos/.

For the second comment, these two changes are separate. This one deals with the perf-test tool, 583 is about kubelet prometheus registration. They might work together to produce the perf results, but lets's not mix them in one PR.